recipes/flask-turbolinks/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "Flask-Turbolinks" %}
+{% set version = "0.2.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "9086fa5d425db3530efe7eb96d90067f0531158864f0e7130846e534069d6381" %}
+
+package:
+  version: '{{ version }}'
+  name: '{{ name|lower }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - flask
+
+test:
+  imports:
+    - flask_turbolinks
+
+about:
+  doc_url: https://github.com/lepture/flask-turbolinks
+  license: BSD
```

Comment:
Sorry -- one final thing and then I'll merge this. Could you change `license: BSD` to `license: BSD-3-Clause`.

Reply:
No problem! `license_family` is correct though?

Reply:
yes